src/strings/en-us.json

```
@@ -1427,6 +1427,7 @@
     "SearchForMissingMetadata": "Search for missing metadata",
     "SearchForSubtitles": "Search for Subtitles",
     "SearchResults": "Search Results",
+    "SearchResultsEmpty": "Sorry! No results found for ",
```

You can use substitutions in translations since some languages may require a different order:

```suggestion
    "SearchResultsEmpty": "Sorry! No results found for \"{0}\"",
```